osi_lane.proto

```diff
@@ -192,69 +192,91 @@ message Lane
+        // 
+        // \attention The points desribing the surface line must be set in the 
         // same ordering (ascending or descending) as the points desribing the 
         // lane boundaries. Example: If the points are deducted from a map format, 
         // the order of points is recommended to be in line with the road coordinate 
         // (e.g. s-coordinate in OpenDRIVE). 
         //
-        // \attention The points describing the center line might be set at
-        // arbitrary distances. When the points are pairwise linearly connected,
-        // the lateral distance to the real ideal line (as used by the
-        // simulation environment internally) must not exceed 5cm. As shown in
-        // the following image:
+        // \attention The points describing the surface line might be set at
+        // arbitrary distances. This allowes to model certain height changes along the 
+        // lane like speed bumps etc. more detailed while still allow very few points 
+        // at parts of the lane where height does not change much.
         //
-        // \image html line_approximation_error.svg "Approximation error"
-        // Approximation error green line.
+        // \note The surface line is the line that describes the height profile 
+        // along the lane. Multiple surface lines can be used to create a certain road shape
+        // if needed. The simplest case would be no surface line for a flat road, since the 
+        // lane boundaries can be used in this case.
+        // 
+        // \attention If more than one surface line exists, they are not allowed to cross each other 
+        // or the lane boundaries, otherwise you could have conflicting height information at these points.
         //
-        // \note The center line is the line that a typical vehicle follows more
-        // or less (depending on the situation, a little more to the left or
-        // right of the center line). The intended direction of travel on the
-        // lane is given by the direction defined by the sequence of points
-        // forming the \c #centerline w.r.t. \c
-        // #centerline_is_driving_direction.
+        repeated Vector3d surfaceline = 3;
+        
+        // The ID of the surface line.
+        // Example: sl4 (see reference picture HighwayExit).
         //
-        // \note The \c #centerline is defined only for \c #type =
-        // \c #TYPE_DRIVING and if exactly one or no \c #lane_pairing pair
-        // exists.
+        // \note Note ID is global unique.
         //
-        // \note Intersections and non-driving lanes do not have a center line.
-        // A vehicle must calculate this individually and depending on the
-        // situation.
+        // \rules
+        // is_globally_unique
+        // \endrules
         //
-        repeated Vector3d centerline = 3;
+        optional Identifier surfaceline_id = 4;
```

Comment:
Because potentially multiple surface lines per lane are envisioned, it wouldn't work like this I think. I think it would make sense to construct a new `message surfaceline{}` with an optional field `Identifier surfaceline_id` and a repeated field `Vector3d surfaceline`. This can message can be used here repeatedly, then. It could also (or instead) be included in the LogicalLanes as you write in the conversation section, so for easier usage it might make sense to put in `osi_common.proto`. What do you think?